api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -86,7 +87,7 @@ WorkspaceACLUpdateResponseList updateWorkspaceACL(
    */
   WorkspaceResponse getWorkspace(String projectName, String workspaceName);
 
-  WorkspaceResponse getWorkspace(DbWorkspace dbWorkspace);
+  Optional<WorkspaceResponse> getWorkspace(DbWorkspace dbWorkspace);
```

Comment:
I don't think this actually ever returns an empty Optional. It will throw an exception if the object DNE.

Reply:
Excellent catch.